### DISCUSSION TITLE
Table Merge

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#434] [Ideas]
Hi Team,

In our cafe, we have most of the tables in 2s and 4s, but somebody comes as a group and want to merge the tables and sit together. And it is the most required option.

So please add the table merge option and so table information and cover information are properly utilized.

**Comment by Blair2004** [comment]
There is an option to merge orders in one.

**Reply by rsidhaarth** [reply]
Yes, I see that, but we are not going to merge orders here. we will first merge the Table and then create one order. As to identify they belong to one group at the starting itself.

**Reply by Blair2004** [reply]
What is the benefit of merging order if it's not to have them billed at once?

**Comment by rsidhaarth** [comment]
An update. I was in search for complete POS for restaurant. And I found that in iRestora they are performing this merge operation by allowing to enter the seat capacity in multiple tables and then create order. Which is like same group using more than one table. And we got table merge.
